docs/misc/faq.md

```diff
@@ -135,8 +135,7 @@ thousands or even millions of containers running in parallel.
 
 ### How do I connect Docker containers?
 
-Currently the recommended way to link containers is via the link primitive. You
-can see details of how to [work with links here](../userguide/dockerlinks.md).
+Currently the recommended way to connect containers is via the Docker network. You can see details of how to [work with links here](https://docs.docker.com/networking).
```

Comment:
"The Docker network feature" is what I'm trying to standardize on. That likely takes it out of the realm of "product."  Other suggestions welcome of course. :grin: